sql/civicrm_data/civicrm_dedupe_rule/HouseholdSupervised.sqldata.php

```diff
@@ -0,0 +1,13 @@
+<?php
+return CRM_Core_CodeGen_DedupeRule::create('HouseholdSupervised')
+  ->addMetadata([
+    'contact_type' => 'Household',
+    'threshold' => 10,
+    'used' => 'Supervised',
+    'title' => ts('Name and Email'),
+    'is_reserved' => 0,
+  ])
+  ->addValueTable(['rule_table', 'rule_field', 'rule_weight'], [
+    ['civicrm_contact', 'household_name', 10],
+    ['civicrm_email', 'email', 10],
```

Comment:
This format again  ☹️ 
I understand not wanting to refactor all of them from the previous PR, but I'm not sure why you're adding new ones. It just feels like introducing oddball patterns into an otherwise very clean new design.
It's your PR so again I won't block it, but again, I don't like it very much. 👿

Reply:
Sorry for being a curmudgeon about this Tim. I can see how the `addValueTable` syntax would look fine to someone who's used to staring at SQL files (header first then data is what INSERT statements look like). But I'm used to staring at API code so I always expect key/value pairs to be on the same line. Even for short lists I find that more readable, and while the compact format is *tolerable* for short lists, I'm biased toward standardization over flexibility after getting burned a few too many times on the overly-tolerant APIv3 code that would accept a wide variety of formats for some params, to the point of making it very difficult to parse or refactor.

Reply:
Related chat: https://chat.civicrm.org/civicrm/pl/db77emj74pyaur65ex4a6sbf3a